core/src/test/scala/pureconfig/ApiSuite.scala

```diff
@@ -128,9 +121,16 @@ class ApiSuite extends BaseSuite {
     loadConfigFromFiles[Conf](files) should failWithType[ThrowableFailure]
   }
 
-  it should "fail if any of the files does not exist" in {
-    case class Conf(f: Float)
-    val files = listResourcesFromNames("/conf/loadConfigFromFiles/priority1.conf") :+ nonExistingPath
-    loadConfigFromFiles[Conf](files) should failWithType[CannotReadFile]
+  it should "ignore files that doesn't exist" in {
+    case class Conf(b: Boolean, d: Double)
+    val files = listResourcesFromNames("/conf/loadConfigFromFiles/priority2.conf") :+ nonExistingPath
+    loadConfigFromFiles[Conf](files) shouldBe Right(Conf(false, 0.001D))
+  }
+
+  "loadConfigWithFallback" should "fallback if no config keys are found" in {
+    case class Conf(f: Float, o: Option[Int], d: Double)
+    val priority1Conf = ConfigFactory.load("conf/loadConfigFromFiles/priority1.conf")
+    // first wo are in priority1.conf, the d is in reference.conf
```

Comment:
Good question... it was copy-pasted from [here](https://github.com/pureconfig/pureconfig/pull/262/files#diff-6e4d5f4a2619b22fc36712e8d1c9ac09L115). I think it means "two". I clarified that comment in 2ab02930ff0733fd5c593241fad84bf3e8f02654.